examples/ssd/caffe2npz.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, model_path):
         super(SSDCaffeFunction, self).__init__(model_path)
 
     def __setattr__(self, name, link):
-        if self.within_init_scope and name != '_within_init_scoep':
+        if self.within_init_scope and instanceof(link, Link):
```

Comment:
Could you rename `link` -> `value`? `Chain.__setattr__` uses this name. (https://github.com/chainer/chainer/blob/master/chainer/link.py#L655)